apps/docs/src/app/platform/component-docs/platform-list/platform-list-examples/platform-list-with-delete-button-example.component.html

```diff
@@ -0,0 +1,45 @@
+<h3>Cozy</h3>
+<h5>Declarative Approach</h5>
+<fdp-list>
+    <fdp-standard-list-item [titleIcon]="'address-book'" title="Title1" [secondayIcons]="[{'icon':'decline'
+    , 'isButton':'true'}]">
+    </fdp-standard-list-item>
+    <fdp-standard-list-item [titleIcon]="'appointment'" title="Title2" [secondayIcons]="[{'icon':'decline'
```

Comment:
should it be `secondaryIcons` instead?

Reply:
Got it, spelling, I will update it. Thank you